### ISSUE TITLE
Install in camunda wildfly

### ISSUE THREAD

**Issue by joaopalma5** [StephenOTT/Camunda-Monitoring#15]
Hello.

I'm trying to install camunda-monitoring in wildfly. And in the first step, I think have to make this https://dzone.com/articles/deploying-spring-boot-app-to-jboss-wildfly but I don't know about kotlin... You can help me?

You can provide more details about compile and install process? please

thanks.

**Comment by StephenOTT** [comment]
Did you figure out the issue?

**Comment by joaopalma5** [comment]
No. You have any idea, how to install this plugin in wildfly version? 

Thanks

**Comment by StephenOTT** [comment]
Do you have to run in wildfly?  Can you run as a docker container as per the docker example?
